api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -123,6 +124,7 @@ public class UserServiceAccessTest {
     DirectoryService.class,
     FireCloudService.class,
     MailService.class,
+    SamApiClientFactory.class,
```

Comment:
Basically anything that imports `AccessTierServiceImpl` has to mockbean this now.